Seminar2/task2.py

```diff
@@ -0,0 +1,21 @@
+num = int(input('Введите число для перевода его в шестнадцатеричную систему: '))
+BASE = 16
+
+
+def convert(number: int, system: int) -> str:
+    temp = list()
+    base_string = 'abcdef'
```

Comment:
тут можно полную строку цифр передать
```suggestion
    base_string = '0123456789abcdef'
```